tools/test.py

```diff
@@ -54,7 +54,15 @@ def parse_args():
         help='tmp directory used for collecting results from multiple '
         'workers, available when gpu-collect is not specified')
     parser.add_argument(
-        '--options', nargs='+', action=DictAction, help='arguments in dict')
+        '--options',
+        nargs='+',
+        action=DictAction,
+        help='custom options to overwrite config')
+    parser.add_argument(
+        '--eval-options',
+        nargs='+',
+        action=DictAction,
+        help='custom options for evaluation')
```

Comment:
The description is hard to be digested by users.

Reply:
Are there any options that cannot be merged by the config?

Reply:
Basically speaking, any options could be merged by config. However, in practice, we just need to change the existing keys.